controller/internal/supervisor/iptablesctrl/acls.go

```diff
@@ -1668,6 +1732,77 @@ func (i *Instance) cleanACLs() error {
 	return nil
 }
 
+// cleanTriremeChains clear the trireme/hostmode chains.
+func (i *Instance) cleanTriremeChains(context string) error {
+
```

Comment:
This is going to constantly give warnings if some of the chains are not there and it will confuse users.

Reply:
These chains always exist. (Trieme-Input/Trireme-Output/Hostmode-Input/Hostmode-Output). All pu specific chains will hang out of the above chains depending on pu type. These chains are removed when enforcer is being stopped/restarted. We shouldnt see warnings. If we do, then iptables isnt in a sane state and needs to be rectified.